### ISSUE TITLE
psd psb format supports

### ISSUE THREAD

**Issue by CrypticCus** [easymodo/qimgv#364]
Can we have psd psb format supports if possible?

thank you.

**Comment by steambug** [comment]
**I support psd, psb ... support!** ;) I work as a graphic designer / illustrator and I use these file formats very often.
Today I spent two hours searching online for software to replace Adobe Bridge and ACDSee. qimgv is the only program which has almost all necessary features and doesn't look ugly - it has a very nice clean and intuitive UI design. And it is incredibly fast, even with large files. To use it for work I only need qimgv to support more file formats and maybe also a menu option or a hotkey to open file in an external editor like Photoshop. But even without these features it's already amazing piece of software and I will keep an eye on it. Thanks!

**Comment by CrypticCus** [comment]
Yup, there are lots of open issues about this software, when they are complete i think this software can be literally the best one out there...

major things missing i saw from opened issues are
more file support as i written
faster image processing on large folders
some cosmetic changes maybe to support new features...

i hope dev can continue after these hard times. we really like this software.
